std/haxe/macro/MacroStringTools.hx

```diff
@@ -48,16 +48,11 @@ class MacroStringTools {
 	/**
 		Tells if `e` is a format string, i.e. uses single quotes `'` as
 		delimiters.
-
-		This only works if `e` has a position which the compiler can find. While
-		this is true for any expressions appearing in real Haxe code (i.e. some
-		.hx file), it might not work for expressions generated by macros.
-
-		This operation depends on the position of `e`.
 	**/
+	@:deprecated("Check for `CString(_,SingleQuotes)` instead")
```

Comment:
Why the deprecation? It is "only" a convenience method that can be achieved with `.match({expr: EConst(CStrong(_, SingleQuotes))})`, but it might as well remain in the API, no?

Reply:
Maybe? It feels a bit awkward to me to have a method specifically for this, but I can live with it :) What do others think?

Reply:
I agree with Aurel.

Reply:
Hmm, also that can/should be done on the haxe side purely, so we can remove `is_fmt_string` from the compiler completely \o/